apps/yapms/src/lib/types/Region.ts

```diff
@@ -39,7 +45,7 @@ export const SavedRegionSchema = RegionSchema.omit({
 		.array()
 });
 
-export const RegionCandidatesSchema = z
+export const SavedRegionCandidatesSchema = z
```

Comment:
Why was this renamed to "SavedRegionCandidatesSchamea"? I don't think "Saved" means anything here.

Reply:
I chose that since the only time this data structure is used is when we are loading candidates in from either the SVG or JSON.